gateway/src/test/java/io/camunda/zeebe/gateway/interceptors/impl/InterceptorRepositoryTest.java

```diff
@@ -189,11 +190,11 @@ void shouldInstantiateInterceptors(final @TempDir File tempDir) throws IOExcepti
     final var interceptors = repository.load(List.of(internalConfig, externalConfig)).instantiate();
 
     // then
-    final var loadedClass = repository.getInterceptors().get("external");
+    final var loadedClass = repository.getInterceptors().get(new InterceptorId("external", 1));
     assertThat(interceptors)
         .hasSize(2)
         .map(interceptor -> (Class) interceptor.getClass())
-        .containsExactlyInAnyOrder(MinimalInterceptor.class, loadedClass);
+        .containsExactly(MinimalInterceptor.class, loadedClass);
```

Comment:
Note for the fix of #15677: A new regressions test case wasn't necessary. We only need to change this line to ensure that the interceptor order is checked.